src/admin/content-block/hooks/useContentBlocksByContentId.ts

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from 'react';
+
+import { fetchContentBlocksByContentId } from '../content-block.services';
+import { ContentBlockSchema } from '../content-block.types';
+
+type UseContentBlocksByContentIdTuple = [ContentBlockSchema[], boolean];
+
+export const useContentBlocksByContentId = (id?: string): UseContentBlocksByContentIdTuple => {
+	const [isLoading, setIsLoading] = useState<boolean>(false);
+	const [contentBlocks, setContentBlocks] = useState<ContentBlockSchema[]>([]);
+
+	useEffect(() => {
+		if (id) {
+			setIsLoading(true);
+
+			fetchContentBlocksByContentId(Number(id))
+				.then(contentBlockResponse => {
```

Comment:
catch toevoegen met toast, zodat de gebruiker altijd op de hoogte is van errors

Reply:
Moet die hier ook nog komen als die in de service al getoond wordt?

Reply:
ah, nee, sorry had ik niet gezien